addon/components/frost-list.js

```diff
@@ -150,11 +150,12 @@ export default Component.extend({
       })
     }
 
-    $(document).on(`keyup.${this.elementId} keydown.${this.elementId}`, this.setShift.bind(this))
+    this._keyHandler = this.setShift.bind(this)
+    $(document).on(`keyup.${this.elementId} keydown.${this.elementId}`, this._keyHandler)
   },
 
   willDestroy () {
-    $(document).off(`keyup.${this.elementId} keydown.${this.elementId}`, this.setShift.bind(this))
+    $(document).off(`keyup.${this.elementId} keydown.${this.elementId}`, this._keyHandler)
```

Comment:
Any way to add a unit test for this? Like maybe stubbing out `setShift.bind` to return something different in `willDestroy` to make sure it's the instance from before that's passed into `off`?